cmd/juju/commands/main_test.go

```diff
@@ -288,15 +288,14 @@ var commandNames = []string{
 	"add-storage",
 	"add-unit",
 	"add-user",
+	"attach",
 	"agree",
 	"agreements",
-	"attach",
```

Comment:
Why is 'attach' affected by this change? AIUI attach is an alias for 'attach-resource' and shouldn't be affected by touching the romulus dependencies.

Reply:
Good point. Surprised my local unit tests didn't catch this

Reply:
Ah so turns out this is a mistake I made with ordering when restoring agree and agreements, hence why unit tests still pass